### PR TITLE
Remove all environment variables options from run.py and add --envs.

### PR DESCRIPTION
**Pull request by fegin**

### CODE REVIEW

**Comment by fegin** [comment]
@KarlLeell Please check this pull request and merge it. After merging this pull request, please modify run.py so that it also supports RNN.

**Comment by fegin** [comment]
@KarlLeell I'll just merge this pull request since I'll use this branch to do the implementation. Please still review it and create another branch to implement RNN support.

File: benchmark_tests/run.py [hunk 1]
```diff
@@ -115,28 +110,29 @@ def main():
...
+    parser.add_argument('--envs', type=str, nargs='+',
+                        help='Environment variables.',
+                        default=['PYTHONPATH=/home/fegin ',
+                                 'LD_LIBRARY_PATH=/usr/local/cuda/lib64 ',
+                                 'CUDA_VISIBLE_DEVICES=0 ',
+                                 'MXNET_ENGINE_TYPE=NaiveEngine ',
+                                 'MXNET_PREFETCH_ALGORITHM=ComputePrefetch ',
+                                 'MXNET_PREFETCH_STEPS=30 ',
+                                 'MXNET_MEM_MGR_TYPE=CUDA ',
+                                 'MXNET_SWAP_ALGORITHM=NaiveHistory ',
```
Comment (Sotskin) [inline benchmark_tests/run.py:124]: The trailing spaces from line 117 to 124 should be removed.

File: benchmark_tests/run.py [hunk 2]
```diff
@@ -115,28 +110,29 @@ def main():
...
+    # Environment variables settings.
+    # NOTE that if b new environment variable is added, it should also be here.
+    parser.add_argument('--envs', type=str, nargs='+',
+                        help='Environment variables.',
+                        default=['PYTHONPATH=/home/fegin ',
+                                 'LD_LIBRARY_PATH=/usr/local/cuda/lib64 ',
+                                 'CUDA_VISIBLE_DEVICES=0 ',
+                                 'MXNET_ENGINE_TYPE=NaiveEngine ',
+                                 'MXNET_PREFETCH_ALGORITHM=ComputePrefetch ',
+                                 'MXNET_PREFETCH_STEPS=30 ',
```
Comment (Sotskin) [inline benchmark_tests/run.py:122]: The env  var for prefetch steps is actually "MXNET_PREFETCH_STEPS_AHEAD". Now all tests will run with steps=100 (default value in gpu_swap_prefetch.cc). This problem was in the earlier version as well.

**Review by Sotskin — COMMENTED** [review]
There seems to be some typos here

**Pull request merged**